symengine/functions.cpp

```diff
@@ -1065,7 +1065,7 @@ RCP<const Basic> acot(const RCP<const Basic> &arg)
 }
 
 ATan2::ATan2(const RCP<const Basic> &num, const RCP<const Basic> &den)
-    : TwoArgFunction(num, den)
+    : TwoArgBasic<Function>(num, den)
```

Comment:
Use `typedef TwoArgBasic<Function> TwoArgFunction` in the header file. Then there's no need to change these